src/libcollectionstest/bit/vec.rs

```diff
@@ -630,6 +630,110 @@ fn test_bit_vec_extend() {
                                  0b01001001, 0b10010010, 0b10111101]));
 }
 
+#[test]
+fn test_bit_vec_append() {
+    // Append to BitVec, that holds a multiple of u32::BITS bits
+    let mut a = BitVec::from_bytes(&[0b10100000, 0b00010010, 0b10010010, 0b00110011]);
```

Comment:
I wouldn't mind an assert in here to guard against someone changing the repr to be e.g. u64 without fixing this test.